include/seastar/http/internal/content_source.hh

```diff
@@ -40,10 +40,15 @@ namespace internal {
  * */
 class content_length_source_impl : public data_source_impl {
     input_stream<char>& _inp;
+    http::reply* _rep;
     size_t _remaining_bytes = 0;
```

Comment:
Note: size_t is wrong here in case of a hypothetical 32-bit port reading >4GB. Not the fault of this patch of course so needn't be fixed here.